### ISSUE TITLE
injector

### ISSUE THREAD

**Issue by DeeCZe** [Keliixx/ps99#1]
any working injector??

**Comment by Keliixx** [comment]
yes there is, i recommend you to use Delta X and HydroGen, you can look around on youtube

**Comment by Keliixx** [comment]
> any working injector??

yes there is, i recommend you to use Delta X and HydroGen, you can look around on youtube

**Comment by Overdusts** [comment]
yesssssssssssssssssssssssssssssssssssssssssss

**Comment by DeeCZe** [comment]
ok i runned Delta X, but how to start? if im in plaza i got white screen insteed buying or server hooping :(
